src/libServer/JSONConversion.cpp

```diff
@@ -698,7 +698,13 @@ const Json::Value JSONConversion::convertTxtoEthJson(
    } else {
      inputField += callData;
    }
+ } else {
+   // Append extra '0x' prefix iff GetCode() gave empty string
+   if (inputField.empty()) {
+     inputField += "0x";
```

Comment:
Would it be neater to write `std::string inputField = "0x";` on line 686, then remove all the extra `0x`es below?

Reply:
OK, I saved another variable too, I have been causing problems by making things go too fast, but it causes a smile at least.

Reply:
I meant that if we set `inputField` to `0x` when it is first defined above, then we can remove all the conditional adding of `0x`es later.